cmd/core-metadata/res/configuration.toml

```diff
@@ -31,6 +31,7 @@ File = './logs/edgex-core-metadata.log'
   Host = 'localhost'
   Port = 48060
 
+#same changes done in this section need to be applied in core-commands service configuration.toml file(temporary)
```

Comment:
Comment isn't necessary. This could be said for any service requiring persistence, and now core-command is just another one of those. For example, if the customer wants to switch and use `Redis` they have to change the Type and Port in all services supporting persistence.

Reply:
I assumed, that each service could use different database type. Some could use redis, others mongo :) That is why I put these comments in the tolm files and used [Databases."Metadata.Primary"] instead of just [Databases.Primary].
But if this is not that case, i am removing all them